<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ookWidget</class>
 <widget class="QWidget" name="ookWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;OOK settings&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="TsWidget">
       <property name="toolTip">
        <string>symbol duration</string>
       </property>
       <property name="text">
        <string>Ts [s]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="TsInput">
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="fsWidget">
       <property name="toolTip">
        <string>sampling frequence used by encode</string>
       </property>
       <property name="text">
        <string>fs [Hz]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="fsInput">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="fcWidget">
       <property name="toolTip">
        <string>carrier frequence</string>
       </property>
       <property name="text">
        <string>fc [Hz]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="fcInput">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="NbitsWidget">
       <property name="toolTip">
        <string>number of bits to encode</string>
       </property>
       <property name="text">
        <string>Nbits</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="NbitsInput">
       <property name="toolTip">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="generateWidget">
       <property name="toolTip">
        <string>sets whether to generate random bit sequence for encode</string>
       </property>
       <property name="text">
        <string>generate</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="generateInput">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="bitsWidget">
       <property name="toolTip">
        <string>sets input bit sequence to encode</string>
       </property>
       <property name="text">
        <string>bits</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="bitsInput">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
